source/get-started/bare-metal-install-desktop.rst

```diff
@@ -643,16 +645,15 @@ Create partitions per requirements in Table 1.
      - /boot
      - 150MB
 
-   * - ``linux-swap``
-     - swap
-     -
-     - 256MB
-
    * - ``ext[234], XFS, or f2fs``
      - root
      - /
      - *Size depends upon use case/desired bundles.*
 
+.. note:: 
+   
+   A swapfile is generated by default. See `Advanced installation`_
```

Comment:
```suggestion
   A 64MiB swapfile is generated by default. The default size may manually set with the ``--swap-file-size`` command-line option.
```

I don't think it is worth the redirect to the Advanced section there isn't any additional information about the swapfile there. We can add the second sentence about overriding the default swapfile size if you think it is useful to the user.

Reply:
I like your suggestion on adding more details on the `swapfile`.  The link--Advanced installation is not a redirect--but was intended to give users more details on default swapfile and the option to create swap partition if desired. 
I'll defer to your suggestion here.